src/errors.rs

```diff
@@ -40,6 +41,7 @@ impl fmt::Display for BcryptError {
             BcryptError::CostNotAllowed(ref cost) => {
                 write!(f, "Cost needs to be between {} and {}, got {}", ::MIN_COST, ::MAX_COST, cost)
             },
+            BcryptError::InvalidPassword => write!(f, "Invalid password"),
```

Comment:
can we get a note about NULL bytes in there so people don't wonder what's invalid about it?

Reply:
Done